src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -33,14 +33,17 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart =
+                shoppingCartService.getByUser(userService.findByEmail(
```

Comment:
`userService.findByEmail()` can return empty Optional. Get value using `.orElseThrow(() -> new NoSuchElementException("message"));`
fix all similar cases in all controllers where are called  `userService.findByEmail()`